### ISSUE TITLE
Getting warning when using react-final-form-hooks in React 17

### ISSUE THREAD

**Issue by shiwano** [final-form/react-final-form-hooks#82]
### Are you submitting a **bug report** or a **feature request**?

bug report (but actually warning report)

### What is the current behavior?

I get the following warning when using react-final-form-hooks in React 17.

```
warning " > react-final-form-hooks@2.0.2" has incorrect peer dependency "react@^16.8.1".
```

### What is the expected behavior?

No warnings.

### Sandbox Link

None.

### What's your environment?

- react-final-form-hooks: 2.0.2
- OS: maxOS
- Node version: v16.10.0

### Other information

None.

**Comment by tofful** [comment]
+1 The same here. Is there any plan to support React 17 and React 18?  
 Have you solved the warning issue @shiwano ?

**Comment by shiwano** [comment]
> Have you solved @shiwano ?

No, I haven't solved it. There is the warning still, but it's worked without problems for us in React 17.
